src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -746,7 +728,7 @@ public virtual XmlNodeType MoveToContent()
                 {
                     case XmlNodeType.Attribute:
                         MoveToElement();
-                        goto case XmlNodeType.Element;
+                        return NodeType;
```

Comment:
I don't know that it matters here but this may result in less efficient code gen.

Reply:
I think this is not significant